src/view/Input.js

```diff
@@ -0,0 +1,39 @@
+import { Console } from "@woowacourse/mission-utils";
+
+class Input {
+  static #CAR_NAME_INPUT_MESSAGE =
+    "경주할 자동차들의 이름을 쉼표(,)로 구분해서 입력하세요.\n";
+
+  static #MOVE_COUNT_INPUT_MESSAGE =
+    "이동을 시도할 횟수를 숫자로 입력해주세요.\n";
+
+  /**
+   * @async
+   * @param {string} message - 입력 전에 출력할 메시지
+   * @returns {Promise<string>} - 입력받은 문자열
+   */
+  static async #readLine(message) {
+    return Console.readLineAsync(message);
+  }
+
+  /**
+   * 자동차 이름들을 입력받는 메서드
+   * @returns {Promise<string>} - 입력받은 자동차 이름들 (쉼표로 구분)
+   */
+  static async readCarNames() {
+    const carNames = await Input.#readLine(Input.#CAR_NAME_INPUT_MESSAGE);
+    return carNames;
+  }
+
+  /**
+   * 이동 횟수를 입력받는 메서드
+   * @async
+   * @returns {Promise<string>} - 입력받은 이동 횟수
+   */
+  static async readMoveCount() {
+    const moveCount = await Input.#readLine(Input.#MOVE_COUNT_INPUT_MESSAGE);
+    return moveCount;
+  }
+}
```

Comment:
```suggestion
static async readCarNames() {  
    const moveCount = await Input.#readLine(Input.#MOVE_COUNT_INPUT_MESSAGE);
    return moveCount;
```
이 부분도 이렇게 고치는게 어떨까요?
추가적으로 readMoveCount() 메서드와 readCarNames() 메서드의 역할이 똑같은데 이 부분도 합치면 어떨까요?
어떤 부분을 받는지는 리턴 값을 받는 상수명으로만 구분지어도 좋을 것 같습니다.

Reply:
사실 적어주신 suggestion 이 잘 이해되지 않습니다 😅 `readMoveCount`와 `readCarNames`가 결론적으로 입력을 받아서 전달하는 동일한 동작을 하고 있기 때문에 합쳐도 괜찮겠다는 말씀이신가요??
제가 제대로 이해한 것이라면,, 
두 함수의 동작이 같음에도 따로 정의해 준 이유는 출력되는 내용은 모두 View 쪽에서 관리하도록 하고 싶었기 때문입니다. ㅎㅎ
함께 출력하는 입력 메시지를 `readMoveCount()` / `readCarNames()`를 호출하는 Controller 쪽에서 인자로 전달하게 되면 출력되는 내용을 View가 아닌 Controller에서 관리하게 되기 때문에 역할 분리가 좀 모호하다는 생각이 들었거든요.

Reply:
> 사실 적어주신 suggestion 이 잘 이해되지 않습니다 😅 `readMoveCount`와 `readCarNames`가 결론적으로 입력을 받아서 전달하는 동일한 동작을 하고 있기 때문에 합쳐도 괜찮겠다는 말씀이신가요??

맞습니다!

저는 코드의 재사용성을 높이는게 효율성이 좋다고 생각하여 주로 코드를 합칠 생각을 하는 편입니다.
@yoouyeon 님은 코드를 최대한 분리하려는 방식이 처음에는 낯설었는데 확실히 역할 분담에 맞게 코드를 짜기 위해서는 @yoouyeon 님의 방식이 훨씬 옳아보이네요! (3주차 과제를 하면서 깨달았습니다ㅠ)
설계부터 의존성을 낮추는 연습을 해야겠어요.
많이 배워갑니다👍👍